docs/README.md

```diff
@@ -469,23 +462,12 @@ kubectl get events --sort-by=.metadata.creationTimestamp -n openfaas-fn
 ```
 
 ##### Troubleshoot Network Policies
-The NetworkPolicy configuration is designed to work with a Kubernetes IngressController. If you are using a NodePort or LoadBalancer you have to deploy NetworkPolicy below.
-```yaml
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
-metadata:
-  name: gateway
-  namespace: openfaas
-spec:
-  policyTypes:
-    - Ingress
-  podSelector:
-    matchLabels:
-      app: gateway
-  ingress:
-    - from: []
-      ports:
-        - port: 8080
+
+The NetworkPolicy configuration is designed to work with a namespaces `openfaas` and `openfaas-fn`.
+If your are using different namespaces you need to make sure that they're labeled correctly:
+```
+kubectl label ns openfaas role=openfaas-system
```

Comment:
Are these labels in the namespaces.yaml file in the faas-netes repo? If not, could we move them there instead of applying the labels manually?

Reply:
https://github.com/openfaas/faas-netes/blob/master/namespaces.yml

Reply:
I think it's an example for custom namespaces, we could use different names

Reply:
It's like @stefanprodan said